pycaching/cache.py

```diff
@@ -845,6 +863,97 @@ def load_by_guid(self):
 
         self.waypoints = Waypoint.from_html(content, "Waypoints")
 
+        self.log_counts = self._get_log_counts_from_print_page(res)
+
+    def _get_log_counts_from_cache_details(self, soup):
+        """Return a dictionary of all log counts found in the page
+        representation, based on the cache details page.
+
+        :param bs4.BeautifulSoup soup: Parsed html document of the cache details page.
+        """
+        lbl_find_counts = soup.find("span", {"id": "ctl00_ContentBody_lblFindCounts"})
+        log_totals = lbl_find_counts.find("p", "LogTotals")
+
+        # Text gives numbers separated by a lot of spaces, splitting retrieves the numbers.
+        # The values might contain thousand separators, which we have to remove before converting
+        # them to real numbers.
+        values = log_totals.text.split()
+        values = [int(value.replace(",", "").replace(".", "")) for value in values]
+
+        # Retrieve the list of image sources.
+        images = log_totals.find_all("img")
+        types = []
+        for image in images:
+            type = image["src"]  # "../images/logtypes/2.png"
+            type = type.split("/")[-1].split(".")[0]  # "2"
+            type = LogType.from_filename(type)
+            types.append(type)
+
+        # Prevent possible wrong assignments when the list sizes differ for some unknown reason.
+        if not len(values) == len(types):
+            raise errors.ValueError(
+                "Different list sizes getting log counts: {} types and {} counts.".format(
+                    len(types), len(values)))
+
+        # Finally create the mapping.
+        log_counts = {}
```

Comment:
```suggestion
        log_counts = dict(zip(types, value))
```
may be more readable

Reply:
Thanks for the hint. I have changed it in both methods.